pkg/overlay/service_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package overlay
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+
+	proto "github.com/coyle/storj/protos/overlay" // naming proto to avoid confusion with this package
```

Comment:
besides continuous integration, we should probably set https://golang.org/doc/go1.4#canonicalimports up so compilation fails without storj.io